torch/csrc/jit/pickler.cpp

```diff
@@ -667,61 +661,84 @@ OpCode Unpickler::readInstruction() {
     case OpCode::GLOBAL: {
       // Module name, it's not needed for anything
       auto module_name = readString();
+      auto class_name = readString();
       // TODO [unpickler refactor] __main__ isn't used by the pickler anymore
       if (module_name == "__main__") {
```

Comment:
You can delete the whole `if (module_name == "__main__") {` block and the comment above as well as long as it passes in fbcode

Reply:
Ill land this and then do that as a PR whose only purpose is to see if we can drop the old pickler stuff.